### PR TITLE
Fuzz Tests for RTVS side message protocol

### LINKED ISSUE #2660: Fuzz Test RTVS side message protocol

### PR DESCRIPTION
**Pull request by karthiknadig**
Fixes #2660

### CODE REVIEW

**Comment by AlexanderSher** [comment]
👍

File: src/Host/Protocol/Impl/Message.cs [hunk 1]
```diff
@@ -36,37 +37,42 @@ public class Message {
             Blob = blob ?? new byte[0];
         }
 
-        public Message(byte[] data) {
+        private Message() { }
+
+        public static Message Create(byte[] data) {
```
Comment (int19h) [inline src/Host/Protocol/Impl/Message.cs]: What is the benefit of having a factory method here rather than constructor?

Comment (karthiknadig) [inline src/Host/Protocol/Impl/Message.cs]: It was just a style choice. I did not want to put parsing code in the constructor.

Comment (int19h) [inline src/Host/Protocol/Impl/Message.cs]: That's a good point. But let's call it `Parse` then.

**Pull request merged**